runtime/src/accounts_background_service.rs

```diff
@@ -122,8 +122,11 @@ impl SnapshotRequestHandler {
 
                 let mut hash_time = Measure::start("hash_time");
                 let mut hash_for_testing = None;
-                snapshot_root_bank
-                    .update_accounts_hash_with_index_option(true, test_hash_calculation);
+                const USE_INDEX: bool = true;
+                snapshot_root_bank.update_accounts_hash_with_index_option(
+                    !USE_INDEX,
```

Comment:
I actually meant to continue the inversion upstream
https://github.com/solana-labs/solana/blob/88f22c360bce341a4aa10564f5f3ae76df8312c7/runtime/src/bank.rs#L4317, https://github.com/solana-labs/solana/blob/master/runtime/src/accounts_db.rs#L3890
So all `do_not_use_index` > `use_index`

But this works for now if that's too much churn for this PR
But

Reply:
My preference would be to change this separately, once the dust settles, and it seems you are ok with that.

Reply:
Yep, works for me.